homeassistant/components/nextcloud/strings.json

```diff
@@ -31,148 +31,280 @@
   },
   "entity": {
     "binary_sensor": {
```

Comment:
One issue with renaming all these strings is that the old ones are removed from lokalise. So even if we rename the keys (which imo looks nicer) the current work on translating this integration is wasted. Recently Paulus could rename some keys in lokalise to make sure the old ones aren't removed, but I think that's something for a future PR. And depending on the amount of work.

Reply:
i've checked it on lokalise, but do not have the permissions to rename or clone existing keys, so i think it is something for a core member with respektive permissions 🤔

Reply:
Yea, I think only Paulus and frenck can do this. What do you think about the intention to keep the current translation keys the same in this PR?

Reply:
to preserve the already spent efforts in translation, we should keep the current names 👍 
will adjust it

Reply:
99e4eb7
the diff still looks "heavy" but this is caused by sorting of the keys

Reply:
Is it just me or are there more strings than before, I am on mobile so I can't easily compare, but I saw Cache Expunges as added strings, but it wasn't there before. Am I missing something or is this PR also adding more sensors?

Reply:
there are more, because there were also additional sensors added

Reply:
Can we move them to a separate PR? I think this PR would be better scoped to use entity descriptions and change device info and add sensors in a later PR

Reply:
7c2c30b